resources/assets/css/coreui4.css

```diff
@@ -222,3 +222,11 @@ h1[bp-section=page-heading] {
     font-size: 2rem;
     line-height: 2.1rem;
 }
+
+html:not([dir=rtl]) .wrapper {
+    padding-left: var(--cui-sidebar-occupy-start, 0)
+}
+
+*[dir=rtl] .wrapper {
```

Comment:
Can we do the same thing without the universal selector? I mean... we can use `*` technically... but I am pretty sure @promatik will cry in the shower tonight, if we do 👀😅